.circleci/config.yml

```diff
@@ -133,6 +133,52 @@ jobs:
       - store_artifacts:
           path: integration-tests/screenshots
 
+  e2e_environment_test:
+    docker:
+      - image: cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+    parallelism: 1
+    parameters:
+      environment:
+        type: string
+        default: dev
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install Dependencies
+          command: npm ci --no-audit
+      - run:
+          name: E2E Check - << parameters.environment >>
+          command: |
+            SPECS=$(circleci tests glob e2e_tests/integration/*.feature | circleci tests split --split-by=timings | tr "\n" "," | tr " " ",")
+            echo "Running feature spec(s): ${SPECS}"
+
+            npx cypress run \
+              --env USERNAME=${CYPRESS_USERNAME_<< parameters.environment >>},PASSWORD=${CYPRESS_PASSWORD_<< parameters.environment >>} \
+              --config-file e2e_tests/cypress.json \
+              --config baseUrl=https://make-recall-decision-<< parameters.environment >>.hmpps.service.justice.gov.uk \
+              --browser chrome \
```

Comment:
Not a major issue at all, but if the image contains Edge browser, that's our main target browser (what probation use on their MoJ laptops) so might be worth using that?

Reply:
Yeah can totally do that 👍